README.md

```diff
@@ -32,50 +32,47 @@ Please make sure that target project has _"typescript"_ installed in _node_modul
-| <code>--tmp [path]</code>                                                                                                                                                                                                                                          | <code>-t [path]</code>                                                                                                                                                        | Directory for storing temporary information (defaults to OS-specific temporary directory). Note that tool completely deletes this folder once finished.                                          |
-| <code>--tsc [options]</code>                                                                                                                                                                                                                                       | <code>-c [options]</code>                                                                                                                                                     | Passed through additional TSC options (defaults to ""). Note that they are not validated or checked for suitability. When passing through CLI it is recommended to surround arguments in quotes **and start with a space** (work-around for a bug in argument parsing dependency of _npm-dts_). |
-| <code>--version</code>                                                                                                                                                                                                                                             | <code>-v</code>                                                                                                                                                               | Output the version number.                                                                                                                                                                       |
-
-<br>
+| Option | Alias | Description |
+|--------|-------|-------------|
+| `--entry [file]` | `-e [file]` | Allows changing main _src_ file from _index.ts_ to something else. It can also be declared as a path, relative to root. |
+| `--force` | `-f` | Ignores non-critical errors and attempts to at least partially generate typings (disabled by default). |
+| `--template` | | Append this template where {0} is replaced with the name/path of the entry module. |
```

Comment:
I would argue it is should be customAlias rather than template and noAlias features. I will add a few ideas at the end about this feature for consideration.